website/index.html

```diff
@@ -61,6 +61,7 @@ <h2>Get started now</h2>
         </div>
         <img src="images/bg_02.png" class="cloud-shell" alt="cloud shell">
       </div>
+      <p class="warning"><b>Warning!</b> Clicking the button will restart your running Cloud Shell instance.</p>
```

Comment:
This wording might be confusing in case users don't currently have a cloud shell instance running and are not familiar with cloud shell. Perhaps, we can reword to make it more clear?

Reply:
How about `*Warning!* If you have a Cloud Shell session running, clicking the button will restart it`? I think linking "Cloud Shell" to https://cloud.google.com/shell might add some clarity, too.

Reply:
Yeah, this is great. Thanks!